app/admin/fiori-service.cds

```diff
@@ -87,6 +96,12 @@ annotate AdminService.GenreHierarchy with {
   ID @UI.Hidden;
 };
 
+@Hierarchy.RecursiveHierarchyActions #GenreHierarchy: {
+  $Type                  : 'Hierarchy.RecursiveHierarchyActionsType',
+  // any name can be the action name with namespace/no bound action name
+  ChangeNextSiblingAction: 'AdminService.moveSiblingAction',
```

Comment:
Maybe rename to `changeNextSibling`. Just a minor thing, but having "Action" in an "Action" name sounds a little weird to me.

Reply:
Ah I meant to rename `moveSiblingAction` to `changeNextSibling`, but `moveSibling` is also fine with me :)